tests/src/3d/testqgstessellator.cpp

```diff
@@ -438,7 +457,10 @@ void TestQgsTessellator::testBackEdges()
   tcNormals << TriangleCoords( QVector3D( 1, 2, 0 ), QVector3D( 1, 1, 0 ), QVector3D( 2, 1, 0 ), up, up, up );
   tcNormals << TriangleCoords( QVector3D( 2, 1, 0 ), QVector3D( 1, 1, 0 ), QVector3D( 1, 2, 0 ), dn, dn, dn );
 
-  QgsTessellator tN( 0, 0, true, false, true );
+  // QgsTessellator tN( 0, 0, true, false, true );
+  QgsTessellator tN = QgsTessellator();
```

Comment:
```suggestion
  QgsTessellator tN;
```
Same in other cases of ` = QgsTessellator()`